lib/zclient.h

```diff
@@ -482,6 +484,9 @@ extern void zclient_send_interface_radv_req(struct zclient *zclient,
 extern int zebra_redistribute_send(int command, struct zclient *, afi_t,
 				   int type, unsigned short instance,
 				   vrf_id_t vrf_id);
+/* Send route notify request to zebra */
+extern int  zebra_route_notify_send(int command, struct zclient *zclient,
```

Comment:
One space between the type and identifier please.

Reply:
@kssoman There is still an extra space here before the `zebra_route_notify_send` :)